doc/scenario.md

```diff
@@ -0,0 +1,317 @@
+# Role Scenario
+
+The role uses a scenario of steps to determine what to do.
+
+Using a scenario of steps, you can:
+- run the specified steps in any order, but API must be followed (with `cartridge_scenario`);
+- replace the steps of the role with your own or add new steps
+  (with `cartridge_custom_steps_dir` or `cartridge_custom_steps`).
+
+## Steps
+
+In `cartridge_scenario` you should specify names of steps. The default scenario includes the following steps:
+- [deliver_package](#deliver_package)
+- [update_package](#update_package)
+- [update_instance](#update_instance)
+- [configure_instance](#configure_instance)
+- [restart_instance](#restart_instance)
+- [wait_instance_started](#wait_instance_started)
+- [connect_to_membership](#connect_to_membership)
+- [edit_topology](#edit_topology)
+- [cleanup_expelled](#cleanup_expelled)
+- [configure_auth](#configure_auth)
+- [configure_app_config](#configure_app_config)
+- [bootstrap_vshard](#bootstrap_vshard)
+- [configure_failover](#configure_failover)
+
+In addition to the steps in the default scenario, there are additional steps:
```

Comment:
In addtition, ... additional...

There are additional steps that aren't included in default scenario, but can be used in the custom one.